src/main/java/nextstep/subway/favorite/application/FavoriteService.java

```diff
@@ -33,8 +34,8 @@ public void createFavorite(LoginMember loginMember, FavoriteRequest request) {
         favoriteRepository.save(favorite);
     }
 
-    public List<FavoriteResponse> findFavorites(LoginMember loginMember) {
-        List<Favorite> favorites = favoriteRepository.findByMemberId(loginMember.getId());
+    public List<FavoriteResponse> findFavorites(LoginMember loginMember, Pageable pageable) {
```

Comment:
```suggestion
    @Transactional(readonly = true)
    public List<FavoriteResponse> findFavorites(LoginMember loginMember, Pageable pageable) {
```
readonly 옵션을 추가해야 slave로 요청합니다.
쓰기 작업은 master로, 읽기 작업은 slave로 해야 성능상 이점이 있을거에요.
mysql innodb는 읽기에서는 락을 걸지 않지만(기본적으로 repeatable read이고 s-lock은 설정해야 걸죠)
쓰기작업시에 gap lock 특성상 상황에 따라 테이블락이 걸릴 수 있어요. (인덱스와 테이블 풀 스캔 등의 개념을 함께 고려해보세요)
그래서 쓰기 작업과 읽기작업을 분리하는 것이 좋습니다.